### PR TITLE
revert(pyproject): add back pytest plugins

### PR DESCRIPTION
**Pull request by aarnphm**
cc @sauyon

Signed-off-by: Aaron <29749331+aarnphm@users.noreply.github.com>

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -236,7 +236,7 @@ exclude = '''
 '''
 
 [tool.pytest.ini_options]
-addopts = ["-rfEX", "-pno:warnings"]
+addopts = ["-rfEX", "-pno:warnings", "-pbentoml.testing.pytest.plugin"]
```
Comment (sauyon) [inline pyproject.toml:239]: Why did you disable warnings here again?

**Review by sauyon — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (aarnphm) [reply pyproject.toml:239]: oh to silent useless warning from tensorflow and flax.

Comment (sauyon) [reply pyproject.toml:239]: Which ones? Can we fix those without silencing warnings globally?

Comment (aarnphm) [reply pyproject.toml:239]: Those warnings aren't related to us. It is a protobuf deprecation on Tensorflow end.

**Pull request merged**